.github/workflows/ms.operationalinsights.workspaces.yml

```diff
@@ -106,8 +106,7 @@ jobs:
       - name: 'Using test file [${{ matrix.moduleTestFilePaths }}]'
         uses: ./.github/actions/templates/validateModuleDeployment
         with:
-          templateFilePath: '${{ env.modulePath }}/deploy.bicep'
-          parameterFilePath: '${{ env.modulePath }}/${{ matrix.moduleTestFilePaths }}'
+          templateFilePath: '${{ env.modulePath }}/${{ matrix.moduleTestFilePaths }}'
```

Comment:
This PR is related to Synapse Privatelink Hubs. Please remove changes to Operational Insights Workspaces and open a different PR for that

Reply:
removed change